tools/appliance/definitions/systemvmtemplate/definition.rb

```diff
@@ -27,15 +27,15 @@
 architectures = {
     :i386 => {
         :os_type_id => 'Debian',
-        :iso_file => 'debian-7.9.0-i386-netinst.iso',
-        :iso_src => 'http://cdimage.debian.org/cdimage/archive/7.9.0/i386/iso-cd/debian-7.9.0-i386-netinst.iso',
-        :iso_md5 => 'e101a11ddb31f85acef542df1a49bf57',
+        :iso_file => 'debian-7.11.0-i386-netinst.iso',
+        :iso_src => 'http://cdimage.debian.org/cdimage/archive/7.11.0/i386/iso-cd/debian-7.11.0-i386-netinst.iso',
+        :iso_md5 => '75055a694508f5b891038ec12d703c9e',
     },
     :amd64 => {
         :os_type_id => 'Debian_64',
-        :iso_file => 'debian-7.9.0-amd64-netinst.iso',
-        :iso_src => 'http://cdimage.debian.org/cdimage/archive/7.9.0/amd64/iso-cd/debian-7.9.0-amd64-netinst.iso',
-        :iso_md5 => '774d1fc8c5364e63b22242c33a89c1a3'
+        :iso_file => 'debian-7.11.0-amd64-netinst.iso',
+        :iso_src => 'http://cdimage.debian.org/cdimage/archive/7.11.0/amd64/iso-cd/debian-7.11.0-amd64-netinst.iso',
+        :iso_md5 => '096c1c18b44c269808bd815d58c53c8f'
```

Comment:
I'm not sure how to do that and it would unnecessary. This file only contains config, think of this as a big properties/config file.

This PR fixes the failing build, the ISO was not available which is why I've fixed it. In the long run, we'll move to packer removing all this code.